### ISSUE TITLE
[Bug] Forced YT Quality on WiFi/Cellular not working + bug when replying to comments

### ISSUE THREAD

**Issue by 1bby** [qnblackcat/uYouPlus#108]
### Device info
- iOS/iPadOS version: 15.4
- Device model: iPhone 13 Pro
- Sideload tool: AltStore
- Version of uYouPlus: 2.1

### Despite selecting the desired automatic quality for videos on the uYouPlus settings, whenever I open any YouTube video the quality selected is always “Auto” rather than 1080p which is what I have selected.

The second issue occurs when replying to comments, the text gets overlapped. This only happens when you reply to any comment under the replies to a main comment. 

![19D7D3E2-2A23-4A21-8876-0B7234DCDBE8](https://user-images.githubusercontent.com/50429732/163660792-13978ccc-da21-4e7c-82ad-fe20faab8281.jpeg)
![0EF3B8AD-47F3-42C0-9417-9A8FB777DA21](https://user-images.githubusercontent.com/50429732/163660794-e9d1b9b9-a103-403d-a624-a702326b0d79.png)
![E9C76C7E-16DA-401F-B463-65F5B04EAA30](https://user-images.githubusercontent.com/50429732/163660797-8282c7a6-ed94-48ed-a86a-544addfc6775.png)

**Comment by qnblackcat** [comment]
When YouTube says `Auto (1080p)`. It actually is playing at 1080p 🤷‍♂️Enable `Stats for nerds` in YouTube settings > General and you will understand

![A2DA160C-1E31-4BAD-98BC-42A7FA39356A](https://user-images.githubusercontent.com/52943116/163662799-21e981e9-9fd3-4bc4-8d38-31c9fd8e7d34.png)
![2E4F9317-0E20-470C-A20F-E552E4BB0EC8](https://user-images.githubusercontent.com/52943116/163662801-315e01b4-4581-4222-be67-06b5e5ada4ad.png)

**Comment by 1bby** [comment]
> When YouTube says `Auto (1080p)`. It actually is playing at 1080p 🤷‍♂️Enable `Stats for nerds` in YouTube settings > General and you will understand
> 
> ![A2DA160C-1E31-4BAD-98BC-42A7FA39356A](https://user-images.githubusercontent.com/52943116/163662799-21e981e9-9fd3-4bc4-8d38-31c9fd8e7d34.png) ![2E4F9317-0E20-470C-A20F-E552E4BB0EC8](https://user-images.githubusercontent.com/52943116/163662801-315e01b4-4581-4222-be67-06b5e5ada4ad.png)

Yeah I get that, but as you can see in the screenshot the (auto) quality was at 480p even though I have 1440p selected in the settings. Most videos do however load at 1080p but just reporting this occasional bug.

**Comment by 1bby** [comment]
I changed the automatic quality to 1080p instead of 1440p in uYou settings and have since not encountered the issue again. Just mainly the comment section bug which needs a fix.

**Comment by qnblackcat** [comment]
Lmk if the issue persists in v17.18.4-2.1

**Comment by qnblackcat** [comment]
Close due to inactive
